p2p/enode/nodedb.go

```diff
@@ -378,6 +390,33 @@ func (db *DB) UpdateFindFails(id ID, ip net.IP, fails int) error {
 	return db.storeInt64(nodeItemKey(id, ip, dbNodeFindFails), int64(fails))
 }
 
+// FindFailsV5 retrieves the discv5 findnode failure counter.
+func (db *DB) FindFailsV5(id ID, ip net.IP) int {
+	return int(db.fetchInt64(v5Key(id, ip, dbNodeFindFails)))
+}
+
+// UpdateFindFailsV5 stores the discv5 findnode failure counter.
+func (db *DB) UpdateFindFailsV5(id ID, ip net.IP, fails int) error {
+	return db.storeInt64(v5Key(id, ip, dbNodeFindFails), int64(fails))
+}
+
+// KeysV5 retrieves discv5 AES keys.
+func (db *DB) KeysV5(id ID, ip net.IP) ([]byte, []byte) {
```

Comment:
Note to self: remove these methods again because they're not used with the new LRU-based session cache